### DISCUSSION TITLE
Is there a way to disable the search functionality?

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#60] [Ideas]
we using this to add options/tags to a record, using an input field, and there won't be options available to preselect.

Is there a way to disable the search so that users can just enter and create options freely?

![image](https://user-images.githubusercontent.com/571649/113243155-27d8ee80-92bb-11eb-8ed9-6e3d08b8c906.png)

**Comment by oyejorge** [comment]
[Would this work for you?](https://jsfiddle.net/sr7ztjn5/)

**Reply by acetinick** [reply]
Perfect)  works great,  thanks!
